tdclient/api.py

```diff
@@ -638,6 +638,14 @@ def _read_tsv_file(self, file_like, **kwargs):
 
 
 def normalized_msgpack(value):
```

Comment:
Do you think if it's a good idea to move `normalized_msgpack` function to `tdclient.util`?

Reply:
agreed 6fd3a6f

Reply:
Note: Need to be fixed after releasing new tdclient https://github.com/treasure-data/pytd/blob/master/pytd/writer.py#L11